src/tasks/tasks.py

```diff
@@ -455,6 +455,87 @@ def update_metrics(self, logits, labels, tagmask=None):
         self.scorer2(logits, labels)
         return
 
+@register_task('npi', rel_path='CoLA_probing_npi/')
```

Comment:
If the code for these two tasks are identical to CoLA, it may not be necessary to make duplicate classes. Just add  "@register_task('npi', rel_path='CoLA_probing_npi/')" with different task name and rel_path should be enough. See mnli-*** as an example.

Reply:
It may also be a good idea to do it with Jason's new subtask